### PR TITLE
Added feature to show array of pending tasks

### PR DESCRIPTION
**Pull request by PaudelaGC**
(code done by Laura, commit done by Pau)

### CODE REVIEW

File: app/helpers/inquirer.js [hunk 1]
```diff
@@ -93,5 +94,20 @@ async function createNewTask(taskController, user) {
     }
 }
 
+async function showPendingTasks(taskController) {
+    let exit = false;
+    const tasks = await taskController.getPendingTasks();
+    let taskArray = [];
+    taskArray = tasks.tasks;
```
Comment (razondpro) [inline app/helpers/inquirer.js]: no need to create a new array..

File: app/helpers/inquirer.js [hunk 2]
```diff
@@ -93,5 +94,20 @@ async function createNewTask(taskController, user) {
...
+    taskArray.forEach(task => {
+        pendingTasks.choices.push(task.title);
+    })
+    pendingTasks.choices.push('back');
+    while (!exit) {
+        const menuOption = await inquirer.prompt([pendingTasks]);
+        //To-do
```
Comment (razondpro) [inline app/helpers/inquirer.js]: finish implementation of exit(back) option in this PR

File: app/tasks/task.list.js [hunk 3]
```diff
@@ -9,6 +9,18 @@ class TaskList {
     append(task){
         this.tasks.push(task)
     }
+
+    getPendingTasks(){
+        return this.tasks.filter(task => task.status === 'pending');
+    }
+
+    getStartedTasks(){
```
Comment (razondpro) [inline app/tasks/task.list.js]: Dont implement unnecesary codes.

File: app/tasks/task.list.js [hunk 4]
```diff
@@ -9,6 +9,18 @@ class TaskList {
...
+        return this.tasks.filter(task => task.status === 'started');
+    }
+
+    getCompletedTasks(){
```
Comment (razondpro) [inline app/tasks/task.list.js]: Dont implement this code for this user story. We are implementing only "pending" task. Leave this code for its own user story (finished/completed) task.

**Comment by PaudelaGC** [comment]
Outdated pull, changes performed and pushed in another commit

**Pull request closed without merging**